### ISSUE TITLE
Project logo

### ISSUE THREAD

**Issue by matteodelabre** [matteodelabre/toltec#194]
We should get a logo for Toltec to make the project easily identifiable. It could be used, for example, on this repo’s README, as the GitHub org’s picture, on the future homepage (see #144) or in the [Toltec installer](https://github.com/Eeems/toltec-installer).

I’m not that much of a graphic designer myself. I’m not opposed to paying a professional designer, but I don’t have a big budget for that (considering this is “only” a side-project). If anyone has suggestions or recommendations, please feel free to comment below.

**Background.** “Toltec” is a not-so-subtle reference to Xochitl, the proprietary reading/drawing/note-taking app distributed with the reMarkable. [Xochitl was a queen of the Toltec civilization from 877 to 916.](https://en.wikipedia.org/wiki/Xochitl_(Toltec)) Xochitl (the person) is part of Toltec (the culture) just like Xochitl (the app) is the kind of object that Toltec (the repository project) aims to distribute.

**Some ideas.** The Toltec temples are perhaps one of the most iconic representations of that culture (see for example the [temples at Tula](https://commons.wikimedia.org/wiki/Category:Piramide_C_(Tollan-Xicocotitlan)), or the [Mayan temples at _Chichen Itza_](https://commons.wikimedia.org/wiki/Category:Chichen_Itza_-_Castillo) which look similar). They can also be stylized easily and could work at different sizes. If you have other ideas, please submit them!

**Comment by Eeems** [comment]
What's the budget you currently have? I'd be willing to chip-in for a professionally created one.

**Comment by LinusCDE** [comment]
From discord:

![grafik](https://user-images.githubusercontent.com/22298664/103901190-1b4c5c00-50f9-11eb-963c-f70b8beabe53.png)
![grafik](https://user-images.githubusercontent.com/22298664/103901211-20111000-50f9-11eb-93aa-900ab64a77dc.png)

**Comment by LinusCDE** [comment]
Asking the community is a great option.

Some of the advantages/disadvantages I see currently. (Feel free to edit more into it!)

Asking Community:

| Advantages | Disadvantages |
| ------------ | -------------- |
| Probably free of charge | May cause an unwanted early communit rollout where the project is buggy and may reduce potential use
| Promotes artists in the community :heart: | Could take longer than we want before release is planned

Paying a professional to do it:

| Advantages | Disadvantages |
| ------------ | -------------- |
| More control over design | Seems counter community
| No premature leak | Finished logo maybe also not done for community rollout

How about, we look for someone on discord who can make a draft, use this for the community rollout and later replace it with an improved community made version. I think that many on discord who check the dev channels would be aware of it anyway. There are probably also some with at least some drawing skill to make such a draft.

I personally am not strongly against and possibility (paid professional, community made, internal draft + community). Though we should probably get to a **conclusion by monday or so** if we want to go forward in a reasonable pace with this.

I would also be willing to chip in with the payment should we decide to go a route that needs that, I don't have much constraints there.

**Comment by LinusCDE** [comment]
I think we can consider this issue closed with the logo done for the webpage:

![logo_85cd2a2b](https://raw.githubusercontent.com/toltec-dev/toltec/85cd2a2bf85813cb32d141e867ab0bf7eef18c33/web/logo.png)

While having a smaller logo would be neat, I think we could address that in a separate issue once the community rollout is done.

**Comment by Eeems** [comment]
@LinusCDE that logo doesn't work well for a favicon or GitHub organization picture though. I thought that's what this issue was also tracking?
If we are closing this to deal with the other parts later, a new issue should be opened and linked.

**Comment by LinusCDE** [comment]
That's a good point. I was going through the issues related to the community rollout and it seems that even though a fav/project/orgination icon is still a good point, for the community rollout it doesn't have any importance imo since most wont see it there anyway.

This issues main point also was on a finding a logo in the first place so I don't think integrating it everywhere is out of the scope of this issue anyway.

Feel free to reopen it though if you feel otherwise.
